src/controllers/CommentController.js

```diff
@@ -0,0 +1,69 @@
+import { comments, users } from '../database/models';
+import Customize from '../helpers/Customize';
+
+
+/**
+ * @exports
+ * @class CommentController
+ */
+class CommentController {
+  /**
+ * users or managers can comment on a travel request
+ * @static
+ * @description POST /api/users/:tripRequestId/comment
+ * @param {object} req request object
+ * @param {object} res response object
+ * @memberof CommentController
+ * @returns {object} data
+ */
+  static async createComment(req, res) {
+    try {
+      const { tripRequestId } = req.params;
+      const { id } = req.user;
+      const { comment } = req.body;
+      const userComment = await comments.create({
+        userId: id,
+        tripRequestId,
+        comment
+
+      });
+      if (userComment) {
+        return Customize.successMessage(req, res, 'Your comment was posted successfully', comment, 201);
+      }
+    } catch (e) {
+      return Customize.errorMessage(req, res, 'Server error', 500);
+    }
+  }
+
+  /**
+* users or managers can view comment thread
+* @static
+* @description POST /api/users/:tripRequestId/comments
+* @param {object} req request object
+* @param {object} res response object
+* @memberof CommentController
+* @returns {object} data
+*/
+  static async getComments(req, res) {
+    try {
+      const { tripRequestId } = req.params;
+      const tripComments = await comments.findAll({
+        attributes: ['comment', 'updatedAt'],
+        include: [{
+          model: users,
+          attributes: ['firstName', 'lastName']
+        }],
+        where: {
+          tripRequestId,
+        }
+      });
```

Comment:
@victorkarangwa4  We can refactor this query and store it in services and probably in Comment service class.